cpp/src/Slice/Parser.h

```diff
@@ -654,14 +654,13 @@ namespace Slice
     {
     public:
         //
-        // Note: The order of definitions here *must* match the order of
-        // definitions of Ice::OperationMode in slice/Ice/OperationMode.ice!
+        // Note: The values of enumerators here *must* match the values for
+        // enumerators of Ice::OperationMode in slice/Ice/OperationMode.ice!
         //
         enum Mode
         {
-            Normal,
-            Nonmutating,
```

Comment:
I deleted the enumerator because it was completely unused.
If we'd prefer to keep this more in sync, I can add it back, but with a `[[deprecated]]` attribute.